### PR TITLE
TKC Portico68 v2: Initial support

### PR DESCRIPTION
**Pull request by TerryMathews**
## Description

Initial support for Portico68 v2

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [X] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [X] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [X] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/tkc/portico68v2/portico68v2.c [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    else {
+         if (rgb_matrix_get_flags() == LED_FLAG_NONE) {
+            for (uint8_t i = 0; i < DRIVER_1_LED_TOTAL; i++) {
+                rgb_matrix_set_color(i, 0, 0, 0);
+            }
+        }
+    }
+	if (!rgb_matrix_is_enabled()) {
+		rgb_matrix_driver.flush();
+    }
```
Comment (drashna) [inline keyboards/tkc/portico68v2/portico68v2.c:146]: This may not be needed.

Comment (TerryMathews) [reply keyboards/tkc/portico68v2/portico68v2.c:146]: When I originally wrote the code for the RGB Matrix version of the original Portico PCB, it was needed if the RGB was disabled via keycode.

Comment (drashna) [reply keyboards/tkc/portico68v2/portico68v2.c:146]: well, that's what I mean. using the flags and such, you shouldn't need the flush, as that is still done by the core code.

Comment (TerryMathews) [reply keyboards/tkc/portico68v2/portico68v2.c]: OK, updated.

**Review by drashna — APPROVED** [review]

File: keyboards/tkc/portico68v2/rules.mk [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = yes         # Virtual DIP switch configuration
```
Comment (fauxpark) [inline keyboards/tkc/portico68v2/rules.mk:10]:
```suggestion
BOOTMAGIC_ENABLE = yes         # Enable Bootmagic Lite
```

File: keyboards/tkc/portico68v2/rules.mk [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+MOUSEKEY_ENABLE = yes          # Mouse keys
+EXTRAKEY_ENABLE = yes          # Audio control and System control
+CONSOLE_ENABLE = no            # Console for debug
+COMMAND_ENABLE = yes           # Commands for debug and configuration
+NKRO_ENABLE = yes              # USB Nkey Rollover
```
Comment (fauxpark) [inline keyboards/tkc/portico68v2/rules.mk:15]:
```suggestion
NKRO_ENABLE = yes              # Enable N-Key Rollover
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by TerryMathews** [comment]
@Collaborators Can I get a final review on this?

**Review by fauxpark — APPROVED** [review]

**Comment by TerryMathews** [comment]
@fauxpark @drashna I think this is ready to be merged?

**Comment by drashna** [comment]
Thanks!

**Pull request merged**